packages/app/src/containers/SymbolInspector.tsx

```diff
@@ -0,0 +1,97 @@
+import { Selectors } from 'noya-state';
```

Comment:
We should rename this file to `SymbolMasterInspector`, since we're also going to have a `SymbolInstanceInspector`